CHANGELOG.rst

```diff
@@ -1,3 +1,25 @@
+**3.0.0 - 08/13/24
+
+Breaking changes:
+  - Update components to use new Vivarium lookup table configuration
+  - Refactor Risk component to create exposure column in state table
+  - Update Observers to use new Vivarium results interface classes and methods
+  - Change the "all_causes" disability weight pipeline from "disability_weight" to "all_causes.disability_weight"
+
+Major changes:
+  - Enable the setting of CSMR using a causes configuration parser
+  - Create cause model and LBWSG risk exposure lookup tables via the configuration
+  - Implement a PublicHealthObserver to help enforce the existence of "measure", "entity_type", "sub_entity", and "entity" columns in the results
+  - Add support for non-loglinear relative risks
+  - Automatically exclude "not_dead" and "no_transition" from cause of death and disease transition results, respectively
```

Comment:
This isn't a change, as we didn't previous record these, so I don't think this should be included here.